mmdet/core/evaluation/eval_hooks.py

```diff
@@ -22,7 +59,48 @@ def _do_evaluate(self, runner):
             self._save_ckpt(runner, key_score)
 
 
-class DistEvalHook(BaseDistEvalHook):
+class DistEvalHook(EvalHook):
```

Comment:
Can this hook support distributed training?

Reply:
Yes